tsconfig.tsbuildinfo

```diff
@@ -0,0 +1,830 @@
+{
```

Comment:
Should this file be .gitignored?

Reply:
Yes you are right, I'll add that to the file